dashboard/test/ui/features/support/connect.rb

```diff
@@ -32,7 +32,8 @@ def saucelabs_browser(test_run_name)
     idleTimeout: 60,
     seleniumVersion: Selenium::WebDriver::VERSION
   }
-  sauce_options[:tunnelIdentifier] = CDO.circle_run_identifier if CDO.circle_run_identifier
+  tunnel_name = CDO.circle_run_identifier || CDO.saucelabs_tunnel_name
```

Comment:
We only ever expect `saucelabs_tunnel_name` to be used in the context of using `sc` for local testing, right? If so, could we add a comment to that effect? Specifically, something that explains to future code archaeologists why we're considering it as an alternative to `circle_run_identifier` here, but not on line 31 above.